frontend/src/pages/PageTimings.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <ContentLayout />
```

Comment:
Not quite sure if you need to use content layout twice, since parent page also uses it (ensure that it won't cause any layout issues

Reply:
the same structure used for  visitors, I think we can try delete this component in future with same real components on this page

Reply:
Same structure doesn't mean that it's right structure)